decidim-initiatives/spec/cells/decidim/initiatives/initiative_m_cell_spec.rb

```diff
@@ -23,52 +23,56 @@ module Decidim::Initiatives
       let(:show_space) { false }
 
       it "renders the card" do
-        expect(subject).to have_css(".card--initiative")
+        skip "REDESIGN_PENDING - Deprecated remove the entire file once fully enabled redesign and removed the m-card"
```

Comment:
since we have enabled the redesign variable, maybe we could cleanup ?

Reply:
Done, I've removed the old cell and adapted the tests